### PR TITLE
feat: make preview work with unity engine

### PR DESCRIPTION
**Pull request by fmiras**
Johny la gente esta muy loca 🎉

### CODE REVIEW

File: static/preview.html [hunk 1]
```diff
@@ -47,4 +55,6 @@
       })
     </script>
   </body>
+  <script src="@/unity/Build/UnityLoader.js"></script>
```
Comment (menduz) [inline static/preview.html]: Please start the URL with / to make it relative to the document.origin

File: packages/unity-interface/dcl.ts [hunk 2]
```diff
@@ -175,37 +184,60 @@ class UnityParcelScene extends UnityScene<LoadableParcelScene> {
...
-      unityInterface.SetPosition(newPosition.x, newPosition.y, newPosition.z)
-    },
-    onLoadParcelScenes: lands => {
-      unityInterface.LoadParcelScenes(
-        lands.map($ => {
-          const x = Object.assign({}, ILandToLoadableParcelScene($).data)
-          delete x.land
-          return x
-        })
+  if (!PREVIEW) {
```
Comment (menduz) [inline packages/unity-interface/dcl.ts]: We should always initialize the UI

File: packages/unity-interface/dcl.ts [hunk 3]
```diff
@@ -175,37 +184,60 @@ class UnityParcelScene extends UnityScene<LoadableParcelScene> {
...
+    await initializeDecentralandUI()
+  }
+
+  if (PREVIEW) {
+    global['handleServerMessage'] = function(message: any) {
+      if (message.type === 'update') {
+        // tslint:disable-next-line: no-floating-promises
+        loadPreviewScene()
+      }
+    }
```
Comment (menduz) [inline packages/unity-interface/dcl.ts]: can we move this id inside loadPreviewScene?

File: packages/unity-interface/dcl.ts [hunk 4]
```diff
@@ -175,37 +184,60 @@ class UnityParcelScene extends UnityScene<LoadableParcelScene> {
...
+    // Warn in case wallet is set in mainnet
+    if (net === ETHEREUM_NETWORK.MAINNET && DEBUG && !AVOID_WEB3) {
+      const style = document.createElement('style')
+      style.appendChild(
+        document.createTextNode(
+          `body:before{content:'You are using Mainnet Ethereum Network, real transactions are going to be made.';background:#ff0044;color:#fff;text-align:center;text-transform:uppercase;height:24px;width:100%;position:fixed;padding-top:2px}#main-canvas{padding-top:24px};`
+        )
       )
+      document.head.appendChild(style)
     }
```
Comment (menduz) [inline packages/unity-interface/dcl.ts]: can we move this to the `preview.ts` entry point? I feel this has nothing to do with unity

**Review by menduz — CHANGES_REQUESTED** [review]

File: packages/unity-interface/dcl.ts (hunk 3 above)
Comment (fmiras) [reply packages/unity-interface/dcl.ts]: I didn't get you, what do you mean?

Comment (menduz) [reply packages/unity-interface/dcl.ts]: sorry, I meant this `if`

**Review by menduz — APPROVED** [review]

**Pull request merged**